### DISCUSSION TITLE
PyCon India 2024 Keynote Suggestions

### DISCUSSION THREAD

**Discussion by palnabarun** [pythonindia/inpycon2024-tasks#25] [Announcements]
Please share your keynote speaker suggestions here. It should contain a brief description about the speaker and include a link to the bio of the speaker. The link could their work, their online profiles etc.

If you like a speaker suggestion, please vote by writing a line about your vote for e.g. "I would also love to hear X as a keynote speaker [because of these Y reasons]". 

The selected keynote will be a mix of Indian and International speakers, so feel free to suggest from both.

We will start reaching out to keynote speakers in the order of preference within the next few weeks. Please suggest your keynote speakers by March 10th 23:59 IST.

The PyCon India 2024 team also encourage, using the reactions in the Github comments to show your support. This will help make a better decision with the keynotes.

**Comment by Kludex** [comment]
I would love to hear @patrick91 as a keynote speaker because he is the president of one of the most famous Python conferences (PyCon Italy), and he is the creator of https://github.com/strawberry-graphql/strawberry.

I would love to hear @adriangb as a keynote speaker because he works at https://github.com/pydantic/pydantic, and maintains https://www.starlette.io/, besides he is one of the most knowledge people regarding async and typing.

**Comment by Pradhvan** [comment]
Andrew Godwin [Twitter](https://twitter.com/andrewgodwin) | [Website](https://aeracode.org/)

* Andrew is a Django core developer and has made some significant contributions to Django like migrations, Asynchronous support, and much more.

Carlton Gibson [Twitter](https://twitter.com/carltongibson) | [Website](https://noumenal.es/)

* Carlton has been a long-time Django contributor and Django Fellow. Besides Django, he also helps maintain DRF, Django-Filter, Channels, Crispy Forms, and co-host a weekly podcast Django Chat.
 
Jessica McKellar [Twitter](https://twitter.com/jessicamckellar) | [Website](https://web.mit.edu/jesstess/www/)

* Jessica has been a director of the Python Software Foundation, co-founder for [Zulip](https://zulip.com/), a contributor to [Twisted](https://twisted.org/) and has made significant efforts to increase the diversity of open-source communities.

**Reply by d-pikapika** [reply]
+1 for Jessica McKellar

**Reply by navyad** [reply]
+1 for Jessica McKellar

**Comment by ghoseb** [comment]
I would like to propose the name of Kovid Goyal @kovidgoyal as a keynote speaker.

About Kovid:

* Kovid is the creator and principal developer of [Calibre eBook Manager](https://github.com/kovidgoyal/calibre).
* Kovid is the creator and principal developer of [Kitty Terminal](https://github.com/kovidgoyal/kitty).
* Needless to say, Kovid has made an incredible amount of impact with his two key projects both as an amazing engineer and a maintainer of two highly influential Python projects that have in many ways changed people's lives.
* Kovid holds a PhD in Quantum Computing
* Recent conference talk: https://www.youtube.com/watch?v=y-nFFtQt-Gg

**Comment by gnurenga** [comment]
## We love to listen Vijaykumar's Keynote

Name: Vijaykumar B
Website: http://www.bravegnu.org/
Github: https://github.com/bravegnu
Twitter/X: bravegnu 

## About Vijaykumar B

> He is the author of Embedded programming with gnu tool chain widely used by many developers
[Embedded Programming with the GNU Toolchain](https://bravegnu.org/gnu-eprog/
)

> He is the Organizer for Chennaipy since 2010 to Present

> Most of the talks at Chennaipy monthly was given by Vijay.

> He was also awarded `kenneth Gonsalves` award [1] in PyCon India 2015

> He has wide variety of experience from Hardware to software in Embedded industry.

> He explored python for micro controllers even before micro python, He even wrote application in python that can be used to control Embedded devices.

> He also gave talks and conducted workshops in Previous PyCon India.

> He was also the chair person for PySangamam 2018 and Pycon India 2019, the most coolest person I have come across during the conference. 

> Some of his talks at ChennaiPy[2] not sure about the audio quality but definitely a great speaker and technologists. We would like to listen to his keynote. 

## Reference
[1]. [About Kennet Gonsalves Award ](https://pyfound.blogspot.com/2012/10/kenneth-gonsalves-posthumously-awarded.html#:~:text=The%20award%20is%20planned%20to,community%20and%20educating%20the%20membership.)
[2]. [Introduction State Charts](https://vimeo.com/222096344l)

**Reply by shrayasr** [reply]
+1 for Vijay's keynote. 

Having worked with him on building Chennaipy, I can attest for the solid thought process that he comes with. And I'd love to see that synthesized into a keynote.

**Reply by bhavaniravi** [reply]
+1 for Vijay. He has been the backbone of Chennai's Python community. I know we all can learn from his wisdom

**Reply by tshrinivasan** [reply]
+1 for Vijay. His vast experience on Python and great love towards open source are adorable. He is a perfect speaker for a keynote.

**Comment by felixs88** [comment]
I would like to propose,

Why don't we try for **Guido Van Rossum**? (Just a thought)

**K Lars Lohn**
https://www.linkedin.com/in/k-lars-lohn-1768a3/details/experience/
https://www.youtube.com/watch?v=bSfe5M_zG2s

**Raymond Hettinger**
https://www.youtube.com/watch?v=wiGkV37Kbxk
https://www.youtube.com/watch?v=9zinZmE3Ogk

**Comment by vellankikoti** [comment]
I want to hear from 

1. Matt Harrison 
https://store.metasnake.com/
https://www.linkedin.com/in/panela/

2. Cassie Kozyrkov
https://www.linkedin.com/in/kozyrkov/

3. Ned Batchelder
https://twitter.com/nedbat

**Reply by navyad** [reply]
+1 for  Ned Batchelder

**Comment by souravroy1990** [comment]
I would love to hear from-

Arjan Egges- He is a popular for his youtube channel and his video contents on Python and Rust

[LinkedIn](https://www.linkedin.com/in/arjanegges/)
[Youtube](https://www.youtube.com/ArjanCodes)
[Github](https://github.com/egges)

**Comment by baali** [comment]
I would recommend [Gora Mohanty](https://github.com/gora) He is an active member and contributor of open source communities in NCR from past 15+ years, always up for volunteering in events, contributes with talk selections, organization and logistics.

1. He was part of the team that ran Sarai's floss fellowship program: https://sarai.net/floss-fellowship/ 
2. He is part of [Indlinux](https://www.indlinux.org/) team that extended Indian language support, including localization of interfaces for major distributions
3. He was/is part of ILUG-D and participated in conducting workshops, orientations to OSS projects in various colleges and that's how I was introduced to the OSS community.
4. He continues to volunteers, participate with PyDelhi events, including monthly and [yearly](https://conference.pydelhi.org/index.html) conferences ([1](https://conference.pydelhi.org/2017/volunteers.html), [2](https://www.jnu.ac.in/event/scss/pydelhi2016/), [3](https://conference.pydelhi.org/volunteers.html))

**Reply by Bhupesh-V** [reply]
+1 for Gora Mohanty

**Comment by sayantikabanik** [comment]
Let's make some noise from @bhavaniravi 💜

Where do I begin?
I have known Bhavani for ages. Relentless, passionate and kind are the words that come to my mind.

⭐️ Active Pythonista, brilliant human 
⭐️ Genius all things Apache Airflow and Data Engineering
⭐️ Her newsletter [Tech Tinkering](https://newsletter.bhavaniravi.com/) brings joy to my inbox.
⭐️ Excellent mentor and have been part of numerous community efforts 

**Bhavani on the Internet ✨**

- [Personal Portfolio](https://www.bhavaniravi.com/)
- [GitHub](https://github.com/bhavaniravi)
- [LinkedIn](https://www.linkedin.com/in/bhavanicodes/)
- [X](https://twitter.com/BhavaniRavi_)

**Reply by abhishekmishragithub** [reply]
I second the nomination of @bhavaniravi for keynote speaker! Totally agree with @sayantikabanik - Bhavani's contributions are amazing. She has been an exemplary mentor to many in our community, making a significant and lasting impact, and her engineering and development skills are equally impressive.
Her passion for Python and its community is contagious, she's a great role model for many folks in the community, I have known.

**Comment by archit15singh** [comment]
I would love to hear from: 
Aravind Srinivas (CEO of perplexity AI)

https://www.linkedin.com/in/aravind-srinivas-16051987/

**Comment by nvs-abhilash** [comment]
I would love to hear from Anthony Sottile (@asottile).

He is the creator of [pre-commit](https://github.com/pre-commit/pre-commit), a core contributor of [pytest](https://github.com/pytest-dev/pytest), and one of the maintainers of [tox](https://github.com/tox-dev/tox). Besides, Anthony has an in-depth understanding of Python, and has been educating beginners and intermediate Python programmers for a while.

- Github - [asottile](https://github.com/asottile)
- Youtube - [@anthonywritescode](https://www.youtube.com/@anthonywritescode)
- Twitch - [anthonywritescode](https://www.twitch.tv/anthonywritescode)
- X - [codewithanthony](https://twitter.com/codewithanthony)

**Comment by tankala** [comment]
I would like to suggest these folks and last year they said to check with them for next year(s):
[Mariatta](https://github.com/Mariatta) -  Python core developer, administrator for the Global PyLadies, co-founder of PyCascades, chairing the PyCon US conference for the years 2023-2024

[Wes McKinney](https://github.com/wesm) - creator of [pandas](https://github.com/pandas-dev/pandas), co-creator of [Apache Arrow](http://github.com/apache/arrow) and [Ibis](http://github.com/ibis-project/ibis) and Member of the [Apache Software Foundation](https://apache.org/)

[Łukasz Langa](https://github.com/ambv) - creator of the popular code auto-formatter [Black](https://black.readthedocs.io/), release manager for Python 3.8 and 3.9

[Carlton Gibson](https://github.com/carltongibson) - Django Fellow, maintaining some of the key packages in the Django ecosystem

[Susan Shu Chang](https://www.susanshu.com/) - [Python Software Foundation](https://www.python.org/psf/) contributing member (open source), 5x PyCon worldwide speaker, Author of [Machine Learning Interviews](https://www.oreilly.com/library/view/machine-learning-interviews/9781098146535/)

[Dustin Ingram](https://dustingram.com/) - director of the [Python Software Foundation](https://www.python.org/psf/) and maintainer of [PyPI](https://pypi.org/)

**Comment by vellankikoti** [comment]
I would like to add two more folks 

1. Geetha Manjunath 

https://www.linkedin.com/in/geetha-manjunath-82b8058

Founder & CEO Niramai Helath Analytix

2. Kalyan Prasad 

http://linkedin.com/in/kalyan-prasad-3a647b22

Data Scientist

**Comment by iamaravindsekar** [comment]
Adding a potential keynote speaker here

1. Prof.Dr.Ashok Jhunjhunwala

https://www.ee.iitm.ac.in/~ashok/

https://en.m.wikipedia.org/wiki/Ashok_Jhunjhunwala
